frontend/src/metabase/api/subscription.ts

```diff
@@ -74,6 +76,13 @@ export const subscriptionApi = Api.injectEndpoints({
           idTag("subscription", id),
         ]),
     }),
+    getChannelInfo: builder.query<ChannelApiResponse, void>({
+      query: () => ({
+        method: "GET",
+        url: `/api/pulse/form_input`,
+      }),
+      providesTags: () => provideSubscriptionChannelListTags(),
+    }),
```

Comment:
sadly, we can't reliably tell if slack and/or email are setup via settings, we have to hit this endpoint. 

note: there's an is-slack-token-valid setting that can be true even when there's no slack token saved 🫠